astrology.lic

```diff
@@ -240,12 +240,6 @@ class Astrology
 
     pause 2
     waitrt?
-
-    observed['pools']
```

Comment:
@jandersson Does this mean `observed` is no longer necessary? See line 203.

Reply:
It isn't used anymore, the state of prediction pools is tracked with the `predict all status` command in game.

Reply:
There's a pretty big block of code to assign to it, could you look into if any of it can be removed?

Reply:
Sure thing